auto_nag/scripts/missed_landing_comment.py

```diff
@@ -0,0 +1,69 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from libmozdata.hgmozilla import Mercurial
+
+from auto_nag import utils
+from auto_nag.bzcleaner import BzCleaner
+
+
+def get_csets_from_pushlog(log):
+    csets = []
+    for push in log["pushes"].values():
+        for cset in push["changesets"]:
+            if any(cset["node"][:12] in c["desc"] for c in push["changesets"]):
```

Comment:
you can use `backedoutby` too:
https://github.com/mozilla/relman-auto-nag/blob/293e7b0dd983fd4c6c3e4430953cb4b3bdce3306/auto_nag/utils.py#L449

Reply:
I did look at that, it doesn't actually work as far as I can tell though.

Reply:
@jcristau do you have an example where it doesn't work?

Reply:
That information is just not present in json-pushes.

Reply:
Oh, sorry, I thought this was json-rev.